ceph/ceph/templates/bin/_log_handler.sh.tpl

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# for busybox sh, which is dash
+
+LOG=$1
+DEFAULT_SIZE=$(expr 100 \* 1024 \* 1024)
```

Comment:
can size be parameterized? for cases where quota is (going to be) enforced on emptydir

Reply:
2nd look, there is a SIZE param